.github/workflows/postgres_extension.yml

```diff
@@ -0,0 +1,28 @@
+name: Prometheus Exporter for Postgres
+
+on:
+  pull_request:
+    branches: [ main ]
+    paths: 
+    - '.github/workflows/postgres_extension.yml'
+    - 'extensions/**'
+
+jobs:
+  build_and_test:
+    name: Rust project
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+
+      - name: setup pgx
+        run: cargo install --locked cargo-pgx
+
+      - name: init
+        run: cargo pgx init
```

Comment:
this takes a lot of time because it downloads and compiles several versions of postgres. ideally we could prebuild these, maybe find or build a utility image that has several versions built.

Reply:
on second thought, its only slow compared to what I expected.  The `pgx` project's CI is a ~ 25 minute job, so maybe it isn't going to be a problem: https://github.com/tcdi/pgx/actions